### ISSUE TITLE
NameError: uninitialized constant Class::Date from irb

### ISSUE THREAD

**Issue by tymat** [neo4jrb/neo4j#852]
```

2.2.1 :001 > require 'neo4j'
NameError: uninitialized constant Class::Date
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j/shared/type_converters.rb:7:in `convert_type'
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j/shared/type_converters.rb:171:in `register_converter'
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j/shared/type_converters.rb:153:in `block in included'
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j/shared/type_converters.rb:151:in `each'
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j/shared/type_converters.rb:151:in `included'
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j/type_converters.rb:5:in `include'
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j/type_converters.rb:5:in `<module:TypeConverters>'
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j/type_converters.rb:2:in `<module:Neo4j>'
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j/type_converters.rb:1:in `<top (required)>'
	from /Users/aeufemio/.rvm/rubies/ruby-2.2.1/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/aeufemio/.rvm/rubies/ruby-2.2.1/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/aeufemio/.rvm/gems/ruby-2.2.1@smartwheels/gems/neo4j-5.0.1/lib/neo4j.rb:26:in `<top (required)>'
	from /Users/aeufemio/.rvm/rubies/ruby-2.2.1/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `require'
	from /Users/aeufemio/.rvm/rubies/ruby-2.2.1/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `rescue in require'
	from /Users/aeufemio/.rvm/rubies/ruby-2.2.1/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:39:in `require'
	from (irb):1
	from /Users/aeufemio/.rvm/rubies/ruby-2.2.1/bin/irb:11:in `<main>'2.2.1 :002 >

**Comment by subvertallchris** [comment]
Do `require 'date'` sometime before calling Neo4j.

**Comment by cheerfulstoic** [comment]
Probably we need to `require 'date'` in the gem, but definitely do it in your project @tymat to see if that fixes things first

**Comment by gareth** [comment]
Having had the same issue, I can confirm that requiring 'date' before 'neo4j' resolves the problem

**Comment by cheerfulstoic** [comment]
Awesome, thanks!  I just released 5.0.9.  Please reopen if that doesn't fix the problem.
